pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -105,6 +107,12 @@ func (r *InferenceServiceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, e
 		return reconcile.Result{}, errors.Wrapf(err, "fails to reconcile ingress")
 	}
 
+	//Reconcile modelConfig
+	configMapReconciler := modelconfig.NewModelConfigReconciler(r.Client, r.Scheme)
+	if err := configMapReconciler.Reconcile(isvc, req); err != nil {
+		return reconcile.Result{Requeue: true, RequeueAfter: 10 * time.Second}, err
```

Comment:
controller runtime should be able to handle the retry if err is returned

Reply:
Changed to ```
		return reconcile.Result{}, err

```